apollo-ios/Sources/Apollo/AnyGraphQLResponse.swift

```diff
@@ -0,0 +1,104 @@
+#if !COCOAPODS
+import ApolloAPI
+#endif
+
+/// An abstract GraphQL response used for full and incremental responses.
+struct AnyGraphQLResponse {
```

Comment:
I see that the properties in this struct are `public` but the struct itself is `internal`. That makes those `public` properties not actually `public`. Is this supposed to be `public`, or can we leave it `internal`? If it doesn't need to be public, I'd prefer to leave it internal, but then we should remove the `public` modifiers on the properties (and the functions in the extensions), just for clarity.

Reply:
I'm not sure why they're public but they don't need to be. I'll reduce their permissions scope, that's a good catch!